proxy/src/error.rs

```diff
@@ -80,6 +80,10 @@ pub enum Error {
     #[error("the Project Name '{0}' is invalid")]
     InvalidProjectName(String),
 
+    /// Seed input is invalid.
+    #[error("the seed '{0}' is invalid: {:1}")]
+    InvalidSeed(String, Option<librad::peer::conversion::Error>),
```

Comment:
Again, I think having some separation like a `seed` module would be good where that module has an `Error` type. Trying to move away from dumping _all_ errors in here, but instead composing it of smaller error variants.